src/utils.ts

```diff
@@ -520,4 +522,40 @@ const katexMarkedExtension = {
     }),
 };
 
+// Overwrites the rendering of heading tokens.
+// Since there are possible non ASCII characters in headings,
+// we escape them by replacing them with dashes and,
+// trim eventual dashes on each side of the string.
+export const renderer = {
+  heading(text: string, level: 1 | 2 | 3 | 4 | 5 | 6) {
+    const escapedText = text
+      .toLowerCase()
+      .replace(/[^\w]+/g, "-")
+      .replace(/^-|-$/g, "");
+
+    return `<h${level} id="${escapedText}">${text}</h${level}>`;
+  },
+};
+
 export const markdownExtensions = [emojisMarkedExtension, katexMarkedExtension];
+
+export function twemoji(node: HTMLElement) {
+  const options = {
+    base: "/",
+    folder: "twemoji",
+    ext: ".svg",
+    className: `emoji`,
+  };
+
+  twemojiModule.parse(node, options);
+  afterUpdate(() => {
+    twemojiModule.parse(node, options);
+  });
+  return {
+    update() {
+      console.warn(
+        "Changing twemoji options after the action was mounted is not possible.",
```

Comment:
How could this even happen?

Reply:
8f9cf0d0989e07edaf2e9452977b6e1395934400